jobserver/authorization/admins.py

```diff
@@ -32,8 +33,6 @@ def ensure_admins(usernames):
         sorted_missing = sorted(missing)
         raise Exception(f"Unknown users: {', '.join(sorted_missing)}")
 
-    # reset all users permissions first
-    User.objects.update(is_staff=False, is_superuser=False)
-
-    # update configured users to be admins
-    admins.update(is_staff=True, is_superuser=True)
+    for admin in admins:
+        admin.roles.append(CoreDeveloper)
```

Comment:
Should we have a `SuperUser` role for superusers?

`CoreDeveloper` may still not be able to to *everything*. Naming the role so it's clear it's special seems sensible.

Reply:
Ideally, in fact, these would all check specific permissions, rather than roles - and the `SuperUser` role automatically gets all permissions - but I thought perhaps you wanted to make the change incremental, hence the suggestion here. However, now I think about it, it might save pain in the long run to keep it all permissions-based from the start - WDYT?

Reply:
I don't think we should, no.

We've talked about this!  Having a group of users with, roughly speaking, root access to the platform is not ideal.  Which is why we set it up this way so we can always check the deployment config.  Longer term I would like us to keep the configuration of internal staff but apply more appropriate roles, and then look up permissions at the call site based on those.  However, we're not there yet, and there are quite a few changes to make between now and then.  In the interim the CoreDeveloper role is what we discussed (and you came up with it!), and also what's in the design GDoc.

Reply:
> it might save pain in the long run to keep it all permissions-based from the start

You persuaded me to add `has_role` for exactly this section of the codebase!

Reply:
Based on our call:

* Add a SuperUserRole
* Use it in place of CoreDeveloper to replace is_superuser
* Comment that it is expected to go away, please don't use it further
* Comment that it exists as a transition so we can assign the correct roles to the relevant staff members